README.md

```diff
@@ -138,7 +137,7 @@ in addition to the table-specific fields listed below.
 
 * page_uuid: reference to a Page
 * capture_time: when this snapshot of the Page was acquired
-* uri: path to stored (HTML) data; could be a filepath, S3 bucket, etc.
+* Uri: path to stored (HTML) data; could be a file path, S3 bucket, etc.
```

Comment:
`uri` is actually the name of a field, so it’s important that it’s all lower-case.